### ISSUE TITLE
Are there any plans to integrate WinUI 3 components?

### ISSUE THREAD

**Issue by zikosichi** [yue/yue#141]
Hey @cheng,

In one of your old comments, you mentioned supporting WinUI 3 in Yue. Do you still have any plans for this?

**Comment by zikosichi** [comment]
That's true they always introduce new frameworks, but I think WinUI 3 will stay here for quite a long time.

**Comment by Frityet** [comment]
@zcbenz it seems like its gonna stay, whats the status of this if you don't mind me asking?

**Comment by zcbenz** [comment]
I agree WinUI 3 is going to stay, but I have quite a lot of things piled up in my hand so I don't have a plan for it in near future.
